_cloudstor/01-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Overview
+order: 1
+duration: 5
+---
+
+### What is AARNET CloudStor
+
+CloudStor is an AARNet developed and supported service that enables AARNet customers and the wider community to quickly and securely sync, share and store files using the high-speed AARNet network. AARnet CloudStor provides a easy to use web based dashboard and You can login the dashboard via [Cloudstor dashboard](https://cloudstor.aarnet.edu.au ) and it will give the user access to both CloudStor (storage) and FileSender (ideal for sharing large files). For information about how to use the dashboard, please refer to [CloudStor Getting Started Guide](https://support.aarnet.edu.au/hc/en-us/articles/227469547-CloudStor-Getting-Started-Guide).
```

Comment:
CloudStor is an AARNet developed and supported service

Maybe just say 'CloudStor is an AARNet service'